L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -902,6 +904,8 @@ namespace trklet {
 
     // if true, run a dummy fit, producing TTracks directly from output of tracklet pattern reco stage
     bool fakefit_{false};
+    // if true EDProducer prduces additionaly bit and clock accurate TrackBuilder output
```

Comment:
Change "output" to "EDProduct" in comment?

Reply:
done